cmd/globalAlignment/globalAlignment.go

```diff
@@ -40,6 +70,19 @@ func main() {
 	visualize := align.View(faOne.Seq, faTwo.Seq, aln)
 	fmt.Println(visualize)
 
+	//raven added this block to put visualized alignment output into MSA Fasta
+	visualizeOutput := ">" + faOne.Name + "\n" + strings.Split(visualize,"\n")[0] + "\n" + ">" + faTwo.Name + "\n" + strings.Split(visualize,"\n")[1]
+	outFileName := *faOut
+	outFile,err1 := os.Create(outFileName) //fileio.EasyCreate and other fileio tools should work, but I don't really know how to use them
+	if err1 != nil {
+		log.Fatalf("Write to file failed on step 1\n")
+	}
+	defer outFile.Close()
+	_,err2 := outFile.WriteString(visualizeOutput)
```

Comment:
Really minor stylistic comment, but often you can use just one `err` variable and overwrite it after each error check. In this case if you got an error during `os.Create` then you will fatal. If you make it to `outfile.WriteString` then you know that `err1 == nil` so it would be fine to store the error from `outFile.WriteString` into `err1` for the check on the next line.

Reply:
I see! And the comment already indicates which step it would have failed at if err != nil. Thank you!